spec/test_app/config/application.rb

```diff
@@ -23,8 +23,12 @@
 
 module TestApp
   class Application < Rails::Application
-    # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 6.1
+    case Rails::VERSION::MAJOR
+    when 7
```

Comment:
Do you need this? Isn't the rails version of the test app under your control?

Reply:
Our CI tests gems against ruby 2.6 and 2.7, and bundler installs rails 6 with ruby 2.6 and rails 7 with ruby 2.7.  I think if the gem claims compatibility with two major versions of rails, it should actually be tested against both of them.

Though I think when we've upgraded the apps which use this gem to rails 7, we can drop rails 6 compatibility from the gemspec and from here.

Reply:
Thanks @barrucadu